documentation/DataLoader.md

```diff
@@ -79,3 +79,12 @@ $ docker run --name dataloader --link couchdb \
     -e GPII_COUCHDB_URL=http://couchdb:5984/gpii \
     vagrant-universal scripts/deleteAndLoadSnapsets.sh
 ```
+
+Example provisioning a local CouchDB installation and working with a clone of `universal` code:
```

Comment:
This is a separate example and doesn't fit with the statement at line 59 "Below are two versions of loading couchdb data..." and the following "the first version is .." and "the second version is...".

Might be nice to either change the context or create a separate section for this example.

Also, it's probably a good chance to update "gpii-pouchdb" reference to get this fix for the CI failure into the master.